frontend/src/components/pages/RegistrantExperience/RegistrationSteps/RegistrationFooter.tsx

```diff
@@ -6,21 +6,23 @@ export type RegistrationFooterProps = {
   nextBtnRef: React.RefObject<HTMLButtonElement>;
   currentStep: RegistrantExperienceSteps;
   isCurrentStepCompleted: boolean;
+  registrationLoading: boolean;
   handleStepNavigation: (stepsToMove: number) => void;
 };
 
 const RegistrationFooter = ({
   nextBtnRef,
   currentStep,
   isCurrentStepCompleted,
+  registrationLoading,
   handleStepNavigation,
 }: RegistrationFooterProps): React.ReactElement => {
   const onNextStep = () => {
     if (isCurrentStepCompleted) {
       handleStepNavigation(1);
     } else {
       alert(
-        "Form does not pass validaiton. Please complete all form fields according to requirements.",
+        "Form does not pass validation. Please complete all form fields according to requirements.",
```

Comment:
I was actually hoping to remove the alert. I don't think the alerts are standard practise and the error states of the fields does the same thing

Reply:
How about a toast?